app/actions/accountsActions.js

```diff
@@ -1,8 +1,9 @@
 import { createActions } from 'redux-actions';
 import actionTypes from '../constants/actionTypes';
 
-export const { createAccount, updateAccount, deleteAccount } = createActions(
+export const { createAccount, updateAccount, deleteAccount, performTransfer } = createActions(
   actionTypes.CREATE_ACCOUNT,
   actionTypes.UPDATE_ACCOUNT,
   actionTypes.DELETE_ACCOUNT,
+  actionTypes.PERFORM_TRANSFER,
```

Comment:
redundant action, should use update_account after create_transfer, (hint redux-thunk)